v22.2/restore.md

```diff
@@ -73,6 +73,8 @@ You can control `RESTORE` behavior using any of the following in the `restore_op
 `debug_pause_on`                                                    | `"error" `                                    |  Use to have a `RESTORE` [job](show-jobs.html) self pause when it encounters an error. The `RESTORE` job can then be [resumed](resume-job.html) after the error has been fixed or [canceled](cancel-job.html) to rollback the job. <br><br>Example: `WITH debug_pause_on='error'`
 `incremental_location`<a name="incr-location"></a> | [`STRING`](string.html) | Restore an incremental backup from the alternate collection URI the backup was originally taken with. <br><br>See [Restore incremental backups](#restore-from-incremental-backups) for more detail.
 <a name="new-db-name"></a>`new_db_name`                             | Database name                                 | Rename a database during a restore with `RESTORE DATABASE movr ... WITH new_db_name = new_movr`. The existing backed-up database can remain active while the same database is restored with a different name. <br><br> See [Rename a database on restore](#rename-a-database-on-restore).
+<span class="version-tag">New in v22.2:</span> `schema_only` | N/A | Verify that a backup is valid by running `RESTORE ... schema_only`, which will restore the backed up schema without any user data. See [Backup Validation](backup-validation.html#validate-a-backup-is-restorable) for detail and an example. For specifics around cluster backups, see [Cluster-level backup validation](backup-validation.html#cluster-level-backup-validation).
```

Comment:
Question: why the cluster backup seealso sentence here? Can't we use `schema_only` with all three backup types (db, table, cluster)?